### PR TITLE
adds new chem "grave-digger"

### PR DESCRIPTION
**Pull request by boodaliboo**
adds a new chem that does what normal soda does but its combined into one you get it by combining space cola dr gibb and space-up.
name originates from what my friends growing up called it when you mixed every soda in the dispenser
#### Changelog

:cl:  
rscadd: Added new useless reagant
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -11,6 +11,12 @@
 	results = list(/datum/reagent/lube = 4)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/silicon = 1, /datum/reagent/oxygen = 1)
 
+/datum/chemical_reaction/lube
```
Comment (ynot01) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
/datum/chemical_reaction/graveyard
```

**Comment by boodaliboo** [comment]
Intended feature not bug intended

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -859,4 +859,18 @@
 	quality = DRINK_GOOD
 	color = "#fffafa"
 	taste_description = "cranberry"
-	glass_name = "glass of sprited cranberry"
\ No newline at end of file
+	glass_name = "glass of sprited cranberry"
+	
+/datum/reagent/consumable/graveyard
+	name = "Graveyard"
+	description = "What Happens when you mix all the sodas in the fountain you get this monstorsity"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: description = "What happens when you mix all the sodas in the fountain? You get this monstrosity!"

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -859,4 +859,18 @@
...
+	color = "#dcb137"
+	quality = DRINK_VERYGOOD
+	taste_description = "Tastes like liquid diabetes"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: taste_description = "liquid diabetes"

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 4]
```diff
@@ -859,4 +859,18 @@
...
+	glass_icon_state = "cream_soda"
+	glass_name = "Graveyard"
+	glass_desc = "just looking at is making you want feel sick"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: glass_desc = "Just looking at this is making you feel sick."

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 5]
```diff
@@ -11,6 +11,12 @@
 	results = list(/datum/reagent/lube = 4)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/silicon = 1, /datum/reagent/oxygen = 1)
 
+/datum/chemical_reaction/graveyard
+	name = "Graveyard"
+	id = /datum/reagent/consumable/graveyard
+	results = list(/datum/reagent/consumable/graveyard = 3)
+	required_reagents = list(/datum/reagent/consumable/space_cola, /datum/reagent/consumable/dr_gibb, /datum/reagent/consumable/space_up = 1) //soda
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/recipes/others.dm]: required_reagents = list(/datum/reagent/consumable/space_cola = 1, /datum/reagent/consumable/dr_gibb = 1, /datum/reagent/consumable/space_up = 1)

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]

**Comment by LoliconSlayer** [comment]
Christ do you know what grammar is Bood

**Comment by boodaliboo** [comment]
No and ktl ill impleiment your suggestion in a few days don’t have access to computer for a bit

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 6]
```diff
@@ -859,4 +859,18 @@
...
 	taste_description = "cranberry"
-	glass_name = "glass of sprited cranberry"
\ No newline at end of file
+	glass_name = "glass of sprited cranberry"
+	
+/datum/reagent/consumable/graveyard
+	name = "Grave-Digger"
+	description = "What happens when you mix all the sodas in the fountain? You get this monstrosity!"
+	color = "#dcb137"
+	quality = DRINK_VERYGOOD
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:868]:
```suggestion
	quality = DRINK_GOOD
```
You dont need to be a genius to get this drink

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 7]
```diff
@@ -859,4 +859,18 @@
 	quality = DRINK_GOOD
 	color = "#fffafa"
 	taste_description = "cranberry"
-	glass_name = "glass of sprited cranberry"
\ No newline at end of file
+	glass_name = "glass of sprited cranberry"
+	
+/datum/reagent/consumable/graveyard
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
/datum/reagent/consumable/gravedigger
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 8]
```diff
@@ -11,6 +11,12 @@
 	results = list(/datum/reagent/lube = 4)
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/silicon = 1, /datum/reagent/oxygen = 1)
 
+/datum/chemical_reaction/graveyard
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
/datum/chemical_reaction/gravedigger
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 9]
```diff
@@ -11,6 +11,12 @@
...
+	name = "Grave-Digger"
+	id = /datum/reagent/consumable/graveyard
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/others.dm:16]:
```suggestion
	id = /datum/reagent/consumable/gravedigger
```

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 10]
```diff
@@ -11,6 +11,12 @@
...
+	results = list(/datum/reagent/consumable/graveyard = 3)
```
Comment (alexkar598) [inline code/modules/reagents/chemistry/recipes/others.dm]:
```suggestion
	results = list(/datum/reagent/consumable/gravedigger = 3)
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
It will be done my sire

**Pull request merged**